woocommerce/woocommerce
# Changed county codes for Ireland according to ISO

fixes #19588 

Testing instructions:
1. Go to checkout
2. Pick country Ireland
3. Check billing state option values, these are the changed values:
```
<option value="CO">Cork</option>
<option value="D">Dublin</option>
<option value="G">Galway</option>
<option value="TA">Tipperary</option>
```
4. Check shipping state options, should show the same
